exporter/collector/metrics_test.go

```diff
@@ -307,7 +307,7 @@ func TestHistogramPointToTimeSeries(t *testing.T) {
 	hdp := ts.Points[0].Value.GetDistributionValue()
 	assert.Equal(t, int64(15), hdp.Count)
 	assert.ElementsMatch(t, []int64{1, 2, 3, 4, 5}, hdp.BucketCounts)
-	assert.Equal(t, 12847.600000000002, hdp.SumOfSquaredDeviation)
+	assert.InDelta(t, 12847.6, hdp.SumOfSquaredDeviation, 0.0001)
```

Comment:
Was failing for me on M1, I think this is an appropriate way of asserting on a floating point number that is going through rounding